sap-3-tier-marketplace-image-multi-sid-xscs-md/nestedtemplates/lb-ERS-Linux.json

```diff
@@ -0,0 +1,300 @@
+            "type": "string",
+            "metadata": {
+                "description": "The name of the the load balancer"
+            }
+        },
+        "stackType": {
+            "type": "string",
+            "metadata": {
+                "description": "The stack type of the SAP system."
+            }
+        },
+        "ersASCSInstanceNumber": {
+            "type": "int",
+            "defaultValue": 2,
+            "metadata": {
+                "description": "The instance number of the ABAP ERS instance"
+            }
+        },
+        "ersSCSInstanceNumber": {
+            "type": "int",
+            "defaultValue": 3,
+            "metadata": {
+                "description": "The instance number of the JAVA ERS instance"
+            }
+        },
+        "subnetId": {
+            "type": "string",
+            "metadata": {
+                "description": "The id of the subnet you want to use."
+            }
+        },
+        "nicBackAddressPoolsArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the backaddress pools for the Network Interface creation"
+            }
+        },
+        "frontendIPConfigurationsArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the frontend IP configurations for the Load Balancer creation"
+            }
+        },
+        "backendAddressPoolsArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the backend address pools for the Load Balancer creation"
+            }
+        },
+        "loadBalancingRulesArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the fload balancing rules for the Load Balancer creation"
+            }
+        },
+        "probesArray": {
+            "type": "array",
+            "defaultValue": [],
+            "metadata": {
+                "description": "The array that contains the probes for the Load Balancer creation"
+            }
+        }
+    },
+    "variables": {
+        "lbPrefix": "[concat('lb', padLeft(parameters('ersASCSInstanceNumber'), 2, '0'))]",
+        "lbrulePrefix": "[concat(variables('lbPrefix'), 'Rule')]",
+        "backendPoolERS": "[concat(variables('lbPrefix'), 'BackendPoolERS')]",
+        "loadBalancerFrontendERS": "[concat(variables('lbPrefix'), 'FrontendERS')]",
+        "lbProbePortERS": "[concat(variables('lbPrefix'), 'ProbePortERS')]",
+        "lbProbePortInternalERS": "[add(62100, parameters('ersASCSInstanceNumber'))]",
+        "idleTimeoutInMinutesERS": 30,
+        "loadBalancingRulesASCS": [
+            {
+                "properties": {
+                    "frontendIPConfiguration": {
```

Comment:
simplify and just use resourceId - won't comment on the rest, but please update them